packages/node_modules/@webex/widget-number-pad/src/NumberPad.tsx

```diff
@@ -77,8 +78,37 @@ export const NumberPad = ({
     useGridFocus<HTMLButtonElement>(childrenRef);
     const numberPadRef = useRef<HTMLDivElement>(null);
     const [isNumberPadFocused, setNumberPadFocused] = useState(false); 
+    const [buttonPressStart, setButtonPressStart] = useState(0);
     const NUMBER_PAD_FOCUSED = 'NumberPadFocused';
     const SEARCH_INPUT_FOCUSED = 'SearchInputFocused';
+    const [timeoutID, setTimeoutID] = useState<NodeJS.Timeout>();
+
+    const onPress = (value: string, letters: string | undefined) => {
```

Comment:
Noticed that the letters argument in the onPress function isn't being used. Could we remove it to streamline the function signature?